src/blocks/metablvk/app.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * A simple text editor
+ */
+const editor = document.querySelector('.editor');
+const textArea = document.createElement('p');
+
+// Build text area
+let text = "I'm a simple text editor...";
+textArea.appendChild(document.createTextNode(text));
+textArea.classList.add('text-area');
+// Build editor
+
+editor.appendChild(textArea);
+
+/**
+ * TODO:
```

Comment:
Committing TODOs is bad practice. Please implement the missing features or remove the comments.